recipes/xerces-c/all/conanfile.py

```diff
@@ -151,7 +151,7 @@ def package_info(self):
         if self.settings.os == "Macos":
             self.cpp_info.frameworks = ["CoreFoundation", "CoreServices"]
         elif self.settings.os in ["Linux", "FreeBSD"]:
-            self.cpp_info.system_libs.append("pthread")
+            self.cpp_info.system_libs.extend(["pthread", "nsl", "m"])
```

Comment:
Some folks complained about nsl in system libs because it's not always available, better remove this one. Don't forget that this system libs hook is a hint, but may be wrong or not robust to all cases.